security-analysis/security-analysis-api/src/main/java/com/powsybl/security/results/BranchResult.java

```diff
@@ -34,17 +34,20 @@
 
     private final double i2;
 
+    private final double flowTransfer;
```

Comment:
What is the definition of flow transfer? Javadoc is missing and just with the code, it is not explicit enough. First question: do you need one field or two ? In AC security analysis, there are two flows `p1` and `p2`. What is the definition of flow transfer for pre-contingency situations and for post-contingency situations when you lost more than one line? Other remark: I think that flow transfer must be optional in branch result, is it what you have done ?

Reply:
This is work in progress (I just added the flag sorry).

To anwser your questions, from my comprehension from the dicussions I add with you and @geofjamg :

 > What is the definition of flow transfer? 
 
 The flow percentage from a branch contingency that is added to another branch.

> Javadoc is missing and just with the code

Yes once we know exactly what we are doing I will document it
 
 > do you need one field or two ?
 
 After discussion with @geofjamg it seems a single field is sufficient
 
 > In AC security analysis, there are two flows p1 and p2
 
 Is that relative to a side ? If yes then this mean we actually compute two percentage, one for each side ? This is not clear for me, could you clarify what you have in mind ? Maybe we should discuss it again.
 
> I think that flow transfer must be optional in branch result, is it what you have done ?

No I try to make something with the information I got, if this must be optional in the end it will be.

Reply:
> This is work in progress (I just added the flag sorry).
> 
> To anwser your questions, from my comprehension from the dicussions I add with you and @geofjamg :
> 
> > What is the definition of flow transfer?
> 
> The flow percentage from a branch contingency that is added to another branch.
> 
> > Javadoc is missing and just with the code
> 
> Yes once we know exactly what we are doing I will document it
> 
> > do you need one field or two ?
> 
> After discussion with @geofjamg it seems a single field is sufficient
> 
> > In AC security analysis, there are two flows p1 and p2
> 
> Is that relative to a side ? If yes then this mean we actually compute two percentage, one for each side ? This is not clear for me, could you clarify what you have in mind ? Maybe we should discuss it again.
> 
> > I think that flow transfer must be optional in branch result, is it what you have done ?
> 
> No I try to make something with the information I got, if this must be optional in the end it will be.

So maybe a good start is:
- To focus on side 1, either for monitored branch than for contingency branch ; for me it could be better to call the field explicitly `flowTransfer1`.
- To try to make that field optional.
What do you think?